managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/MetaStoreImpl.java

```diff
@@ -47,11 +53,27 @@
     private final MetadataStore store;
     private final OrderedExecutor executor;
 
+    public static final short MAGIC_MANAGED_LEDGER_INFO_METADATA = 0x0b9c;
+    private CompressionType compressionType = null;
```

Comment:
initiate to be `None` instead of `null`?

Reply:
That's better.